debian/control

```diff
@@ -17,7 +17,7 @@ Build-Depends: autoconf,
                libdw-dev,
                libelf1,
                libelf-dev,
-               libkdumpfile10,
+               libkdumpfile,
```

Comment:
Is this the wrong diff?

Reply:
no.. our is called "libkdumpfile".. ubuntu's is named "libkdumpfile10"..

so we need to revert back to our name..